src/libPMacc/include/particles/tasks/TaskSendParticlesExchange.hpp

```diff
@@ -113,6 +113,15 @@ namespace PMacc
         virtual ~TaskSendParticlesExchange()
         {
             notify(this->myId, RECVFINISHED, NULL);
+            if(retryCounter != 0)
+            {
+                std::cerr << "Send buffer for particle species " <<
+                    ParBase::FrameType::getName() <<
+                    " is to small (max particles per send operation:" << maxSize <<
+                    ", direction:" << exchange <<
+                    ", retried:" << retryCounter <<
```

Comment:
Could you add spaces after the colons so that the values do not collide with the associated names.